controller/scripts/csi_pb2.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -x
+
+version="$1" # currently k8s on v1.2.0
+curl -o csi.proto https://raw.githubusercontent.com/container-storage-interface/spec/"$version"/csi.proto
+python -m grpc_tools.protoc --proto_path=. --grpc_python_out=./controller/csi_general --python_out=./controller/csi_general csi.proto
```

Comment:
![image](https://user-images.githubusercontent.com/57443811/98015372-60a49280-1e05-11eb-911d-ccedbc0295b3.png)
I'm starting to think we need to add a linter for [these gaps](https://github.com/IBM/ibm-block-csi-operator/pull/103#discussion_r507624526)

Reply:
done.
its not tested so if you miss it its hard to see it later..